src/test/performance/logistic_test.cpp

```diff
@@ -0,0 +1,234 @@
+                        "output", "refresh=10000", "file=test/performance/logistic_output.csv"};
+  using boost::posix_time::ptime;
+  using boost::posix_time::microsec_clock;
+
+  ptime time_start;
+  for (int n = 0; n < N; ++n) {
+    time_start = microsec_clock::universal_time(); // start timer
+    stan::common::command<stan_model>(argc, argv);
+    milliseconds_per_run[n] 
+      = (microsec_clock::universal_time() - time_start).total_milliseconds();
+    last_draws_per_run[n] = get_last_iteration_from_file("test/performance/logistic_output.csv");
+  }
+  SUCCEED();
+}
+
+bool close_enough(double expected, double val) {
+  if (std::fabs(expected - val) < 1e-10)
+    return true;
+  return false;
+}
+
+// evaluate
+TEST_F(performance, values_from_tagged_version) {
+  int N_values = 8;
+  ASSERT_EQ(N_values, last_draws_per_run[0].size())
+    << "last tagged version, 2.5.0, had " << N_values << " elements";
+  
+  matches_tagged_version = true;
+  Eigen::Matrix<double, -1, 1> first_run = last_draws_per_run[0];
+  EXPECT_FLOAT_EQ(-67.5276, first_run[0])
+    << "lp__: index 0";
+  matches_tagged_version &= close_enough(-67.5276, first_run[0]);
+
+  EXPECT_FLOAT_EQ(0.773672, first_run[1])
+    << "accept_stat__: index 1";
+  matches_tagged_version &= close_enough(0.773672, first_run[1]);
+
+  EXPECT_FLOAT_EQ(0.901013, first_run[2])
+    << "stepsize__: index 2";
+  matches_tagged_version &= close_enough(0.901013, first_run[2]);
+
+  EXPECT_FLOAT_EQ(2, first_run[3])
+    << "treedepth__: index 3";
+  matches_tagged_version &= close_enough(2, first_run[3]);
+
+  EXPECT_FLOAT_EQ(3, first_run[4])
+    << "n_leapfrog__: index 4";
+  matches_tagged_version &= close_enough(3, first_run[4]);
+
+  EXPECT_FLOAT_EQ(0, first_run[5])
+    << "n_divergent__: index 5";
+  matches_tagged_version &= close_enough(0, first_run[5]);
+
+  EXPECT_FLOAT_EQ(1.71115, first_run[6])
+    << "beta.1: index 6";
+  matches_tagged_version &= close_enough(1.71115, first_run[6]);
+
+  EXPECT_FLOAT_EQ(-0.291085, first_run[7])
+    << "beta.2: index 7";
+  matches_tagged_version &= close_enough(-0.291085, first_run[7]);
+}
+
+TEST_F(performance, values_same_run_to_run) {
+  int N_values = last_draws_per_run[0].size();
+  
+  all_values_same = true;
+  for (int i = 0; i < N_values; i++) {
+    double expected_value = last_draws_per_run[0][i];
+    for (int n = 1; n < N; n++) {
+      EXPECT_FLOAT_EQ(expected_value, last_draws_per_run[n][i])
+        << "expecting run to run values to be the same. Found run "
+        << n << " to have different values than the 0th run for "
+        << "index: " << i;
+      all_values_same &= close_enough(expected_value, last_draws_per_run[n][i]);
+    }
+  }
+}
+
+template <typename T>
+std::string quote(const T& val) {
+  std::stringstream quoted_val;
+  quoted_val << "\""
+             << val
+             << "\"";
+  return quoted_val.str();
+}
+ 
+std::string get_git_hash() {
```

Comment:
Does this mean our tests will only run if someone has Git installed?  Kind of hard to imagine someone running without git, but I also don't see why it should be a dependency here.